project/AppDependencies.scala

```diff
@@ -19,7 +19,7 @@ private object AppDependencies {
       "com.typesafe.play" %% "play-test" % PlayVersion.current % scope,
       "org.pegdown" % "pegdown" % "1.6.0" % scope,
       "org.jsoup" % "jsoup" % "1.13.1" % scope,
-      "uk.gov.hmrc" %% "bootstrap-test-play-28" % "5.14.0" % scope,
+      "uk.gov.hmrc" %% "bootstrap-test-play-28" % "5.20.0" % scope,
       "org.scalacheck" %% "scalacheck" % "1.15.4" % scope,
       "org.scalatestplus" %% "mockito-3-12" % "3.2.10.0" % scope,
       "com.github.tomakehurst" % "wiremock-jre8" % "2.27.2" % scope,
```

Comment:
jsoup -> 1.14.3
wiremock-jre-8 -> 2.32.0
mockito-core -> 4.3.1

Reply:
Vow, thats great Johnny. I will do it.